recipes/default.rb

```diff
@@ -19,6 +19,8 @@
 # limitations under the License.
 #
 
+include_recipe 'chef-sugar'
```

Comment:
The `Platformstack` library uses it, so the gem must be installed before any calls to the library.